manifests/plugin.pp

```diff
@@ -0,0 +1,43 @@
+# common code for the 3rd party plugins
+#
+define collectd::plugin (
+  $config_file_name,
+  $config_template,
+  $manage_package = true,
+  $package_name = undef,
+  $package_version = present,
+  $modules = undef,
+  $filter_metrics = false,
+  $filter_metric_rules = {},
+){
+  validate_string($config_file_name)
+  validate_string($config_template)
+  validate_bool($filter_metrics)
+
+  if $modules {
+    validate_hash($modules)
+  }
+
+  if $manage_package {
+    unless $package_name  {
+      $package_name = $name
+    }
+
+    collectd::check_and_install_package { "${package_name} for ${name} plugin":
```

Comment:
Is there a reason the packages are being bundled in the debs and not in the rpms, why are they not bundles with the same plugin content? seems like this could lead to potential version mismatches with rpms if pinning and not always using latest